### PR TITLE
Update VisualStudioCode.gitignore

### PR DESCRIPTION
**Pull request by martinVyoo**
All of these files are user specific, they should not be included by default
VisualStudioCode will include a functionality in the future to sync the settings separately
https://github.com/microsoft/vscode/issues/2743

# Pull Request

Thank you for contributing to @dvcs/gitignore and https://www.gitignore.io.

## New or update

Select the appropriate check box for this pull request. This helps when merging to ensure there are no conflicts with other templates or misunderstandings of how thee template list works.

### New

- [ ] Template - New `.gitignore` template
- [ ] Composition - Template made from smaller templates
- [ ] Inheritance - Template similar to an existing template
- [ ] Patch - Template extending functionality of existing template

### Update

- [ X] Template - Update existing `.gitignore` template

## Details

The removed files in the gitignore are user/environment specific, they should not be included in the git repository by default, since other contributors of the repository will have their settings being overwritten by what is in the git

VisualStudioCode will include a functionality in the future to separately sync settings https://github.com/microsoft/vscode/issues/2743

But even without that functionality, these files should not be included by default and rather be added by the user if he wants to add them to their repository

### CODE REVIEW

File: templates/VisualStudioCode.gitignore [hunk 1]
```diff
@@ -1,6 +1,2 @@
 .vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
```
Comment (WildCard65) [inline templates/VisualStudioCode.gitignore:4]: These two files should remain included as they're workspace/project specific, NOT user specific.

**Review by WildCard65 — CHANGES_REQUESTED** [review]

File: templates/VisualStudioCode.gitignore (hunk 1 above)
Comment (martinVyoo) [reply templates/VisualStudioCode.gitignore:4]: I think I can agree on this, as long as information written in these files do not contain information specific to a user's environment (in which case they are using these files wrong)

**Review by WildCard65 — APPROVED** [review]
These look good to me, though I'm an outside user for this.

**Comment by gnechita-toptal** [comment]
@martinVyoo Thanks for your contribution.

**Pull request merged**